src/doc/trpl/error-handling.md

```diff
@@ -1,6 +1,6 @@
 % Error Handling
 
-> The best-laid plans of mice and men  
```

Comment:
Haha, good. It was me who added that one some time ago ;)

On Sun, Mar 15, 2015 at 7:26 PM, Tamir Duberstein
<notifications@github.com> wrote:

>> @@ -1,6 +1,6 @@
>>  % Error Handling
>>  
>> -> The best-laid plans of mice and men  
> Heh, this is only instance of this usage. Fixed.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/rust-lang/rust/pull/23385/files#r26450465